src/psyclone/domain/lfric/kern_call_arg_list.py

```diff
@@ -486,10 +486,10 @@ def stencil_2d_max_extent(self, arg, var_accesses=None):
 
         :param arg: the kernel argument with which the stencil is associated.
         :type arg: :py:class:`psyclone.lfric.LFRicKernelArgument`
-        :param var_accesses: optional SingleVariableAccessInfo instance \
+        :param var_accesses: optional AccessSequence instance \
             to store the information about variable accesses.
         :type var_accesses: \
-            :py:class:`psyclone.core.SingleVariableAccessInfo`
+            :py:class:`psyclone.core.AccessSequence`
```

Comment:
This can be a one-liner now (70 width now.

Reply:
Yes, done